jbmc/src/java_bytecode/java_entry_point.cpp

```diff
@@ -65,12 +66,50 @@ static bool should_init_symbol(const symbolt &sym)
   return is_java_string_literal_id(sym.name);
 }
 
+/// Get the symbol name of java.lang.Class' initializer method. This method
+/// should initialize a Class instance with known properties of the type it
+/// represents, such as its name, whether it is abstract, or an enumeration,
+/// etc. The method may or may not exist in any particular symbol table; it is
+/// up to the caller to check.
+/// \return Class initializer method's symbol name.
+irep_idt get_java_class_literal_initializer_signature()
```

Comment:
Perhaps the doxygen comment should document the method that must be implemented (I assume @allredj is aware of this, but tagging just in case).